scripts/area_plot.py

```diff
@@ -0,0 +1,613 @@
+        by `rough_level` to display the data grouped by `rough_level`. The
+        `rough_level` should smaller than the `fine_level`. 
+    samples : list, optional
+        The columns from `table` to be included in the analysis. If `samples`
+        is None, then all columns in `table` except `tax_col` will be used.
+    tax_col : str, optional
+        The column in `table` which contains the taxonomic information.
+    tax_delim: str, optional
+        The delimiter between taxonomic levels, for example "|" or ";".
+    multilevel_table: bool, optional
+        Whether the table contains multiple concatenated, in which cases 
+        considering `nan` will filter the samples to retain only the levels 
+        of interest. This is recommended for kracken/bracken tables, but not 
+        applicable for some 16s sequences
+    abund_thresh_rough, abund_thresh_fine : float [0, 1]
+        The mean abundance threshhold for a taxonomic group to be plotted for
+        the higher level grouping (`abund_thresh_rough`) and sub grouping
+        level. This will be used in conjunction with the `group_thresh_rough`
+        and `group_thresh_fine` to determine the number of groups to be
+        included.
+    group_thresh_fine, group_thresh_rough: int, [1, 6]
+        The maximum number of taxonmic groups to display for the respective 
+        level. If `group_thresh_rough` > 6, then it will be replaced with 
+        6 because this is the maximum number of avaliable color groups.
+
+    Returns
+    -------
+    Figure
+        A 8" x 4" matplotlib figure with the area plot and legend.
+
+    Also See
+    --------
+    single_area_plot
+
+    """
+
+    # Parses the taxonomy and collapses the table
+    taxa = extract_label_array(table, tax_col, tax_delim)
+    if samples is None:
+    	samples = list(table.drop(columns=[tax_col]).columns)
+
+    # Gets the appropriate taxonomic level information to go forward
+    collapsed = level_taxonomy(table, taxa, samples, 
+                               level=np.array([fine_level]), 
+                               consider_nan=multilevel_table)
+    samples = collapsed.columns
+
+    # Gets the top taxonomic levels
+    upper_, lower_, = profile_joint_levels(collapsed, rough_level, fine_level, 
+    								       samples=samples,
+                                           lo_thresh=abund_thresh_rough, 
+                                           lo_count=min(5, group_thresh_rough),
+                                           hi_thresh=abund_thresh_fine,
+                                           hi_count=group_thresh_fine,
+                                           )
+    # Gets the colormap 
+    cmap = define_join_cmap(upper_)
+    upper_.index = upper_.index.droplevel('rough')
+    lower_.index = lower_.index.droplevel('rough')
+
+    # Plots the data
+    fig_ = plot_area(upper_.astype(float), lower_.astype(float), cmap)
+
+    return fig_
+
+
+# Sets up the main arguments for argparse.
```

Comment:
I think you should move the argparse creation into a separate function. That way, the code can be imported cleanly into other tools in the future without having the side effect of creating the argparser. The function can easily be called if the user calls that function. 

e.g.:
```
def create_argparser():
    """Create argparser"""
    parser_one = argparse.ArgumentParser(...)
    .... etc
    return parser_one
```

I hope you understand what I mean, just me me know otherwise @jwdebelius